### ISSUE TITLE
Archives and their collections

### ISSUE THREAD

**Issue by RichardWallis** [schemaorg/schemaorg#1758]
[The Schema Architypes W3C Community Group](https://www.w3.org/community/architypes/) have been working on a proposal for enhancements to Schema.org to enable the description of Archives and their collections/contents.  Which I describe below for consideration.

It consists of two new types:

- [Archive](http://archive.sdo-archive.appspot.com/Archive): Institution with archival holdings. An Archive, or Archives, is an organization which keeps and preserves archival material and potentially makes it accessible to the public.
- [ArchiveComponent](http://archive.sdo-archive.appspot.com/ArchiveComponent): An intangible type to be applied to any archive content, carrying with it a set of properties required to describe archival items and collections. 

and three new properties:

- [accessConditions](http://archive.sdo-archive.appspot.com/accessConditions)
- [archiveHeld](http://archive.sdo-archive.appspot.com/archiveHeld)
- [holdingArchive](http://archive.sdo-archive.appspot.com/holdingArchive)

plus the broadening of the range of four existing properties (_hasPart_, _isPartOf_, _itemCondition_, _location_). 

A test version of the proposal is operating here: [http://archive.sdo-archive.appspot.com/](http://archive.sdo-archive.appspot.com)   The example assumes an 'archive' extension, however the proposal will be for an introduction into 'pending' and thence into the core.

```Archive``` is a subtype of _LocalBusiness_, similar to _Library_.  It has a single new property ```archiveHeld``` for linking to relevant archived items.

```ArchiveComponent``` is the main type in the proposal.  The model behind this proposal separates out the question of ‘what’ something is, from the fact it is in an archive and therefore has a set of archive specific properties related to it.  
Making use of Schema's Multi Type Entity (MTE) capabilities, _ArchiveComponent_ is designed to be used with one or more other types.  Thus the 'what' of a thing is left to existing standard SDO types (e.g. _Collection_, _CreativeWork_, etc.).

This approach is used both for individual items in a collection, and the collections themselves (combining _Collection_ with _ArchiveComponent_).

Examples are available on both the [Archive](ArchiveComponent) & [ArchiveComponent](http://archive.sdo-archive.appspot.com/ArchiveComponent) pages.  For more details (model, background, etc.) see [the proposal Wiki page](https://www.w3.org/community/architypes/wiki/Alternative_1_model_proposal).

A JSON-LD example of a sound recording held in an archive:
```
<script type="application/ld+json">
{
  "@context": "http://schema.org",
  "@id": "https://archiveshub.jisc.ac.uk/data/gb71-thm/407/thm/407/8/3",
  "@type": ["AudioObject","ArchiveComponent"],
  "identifier": "GB 71 THM/407/8/3",
  "name": "Sound Recording of Lines from My Grandafther's Forehead (Radio)",
  "about": "Comedy",
  "description": "Sound recording of the first radio broadcast of Lines from My Grandfather's Forehead by Ronnie Barker and others. Duration: max 90 mins.",
  "inLanguage": "EN",
  "dateCreated": "1971-1972",
  "duration": "PT90M",
  "playerType": "Audio Cassette",
  "accessConditions": "Please check with the Theatre and Performance enquiry team regarding access arrangements before making an appointment to listen to this item.",
  "location": "https://archiveshub.jisc.ac.uk/search/locations/eae30daa-1bf9-33d9-bf1c-7aeb220d2e76",
  "isPartOf": "https://archiveshub.jisc.ac.uk/data/gb71-thm/407/thm/407/8"
}
</script>
```

**Comment by rvguha** [comment]
This looks great!

guha

**Comment by danbri** [comment]
Just checking over the details, I see we have ArchiveComponent as Intangible, ... and yet the PR changes core properties to attach some rather tangible attributes. Most awkwardly: "itemCondition", but also "location"; and it is awkward also making a special case exception for hasPart/isPartOf.

Can we make this a CreativeWork? That type is pretty inclusive already.

For now I think best to comment out the property changes:

```
<!--
<div typeof="rdf:Property" resource="http://schema.org/hasPart">
	<span class="h" property="rdfs:label">hasPart</span>
	<span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
	<span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
</div>

<div typeof="rdf:Property" resource="http://schema.org/isPartOf">
	<span class="h" property="rdfs:label">isPartOf</span>
	<span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
	<span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
</div>


<div typeof="rdf:Property" resource="http://schema.org/location">
	<span class="h" property="rdfs:label">location</span>
	<span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
</div>


<div typeof="rdf:Property" resource="http://schema.org/itemCondition">
	<span class="h" property="rdfs:label">itemCondition</span>
	<span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/ArchiveComponent">ArchiveComponent</a></span>
</div>
-->

**Comment by RichardWallis** [comment]
@danbri There was discussion at the time the proposal was drafted as to if _ArchiveComponent_ should be a CreativeWork or an Intangible subtype.  The reason for making it an Intangible was that not all things in an archive collection are creative works (examples of planes and boxes of toys come to mind).  However, placing something into an archive collection could be potentially considered as a creative act in itself.

In the light of this we could make _ArchiveComponent_  a subtype of _CreativeWork_ as you suggest, without too much controversy.

Fundamental to the definition of an _ArchiveComponent_  are the _hasPart_ & _isPartOf_ properties, as the collection an item is part of is a defining factor of its description.  These would be inherited from CreativeWork if we went that way.

Important to the definition are the ability to identify the condition of an item and its location.  

The nearest match inherited from CreativeWork for _location_ is _contentLocation_ but this is more about what is described/depicted by the CreativeWork, not where it is located.   My suggestion here would be to create an _itemLocation_ property, as a sub-property of _location_, with a domain of _ArchiveComponent_, or potentially _CreativeWork_.

_itemCondition_ is a little more difficult, as its use in the proposal does stretch its definition, and the _OfferItemCondition_ enumeration it expects, somewhat.  An alternative property name and domain does not immediately leap to mind.

To summarise what I am suggesting:
- Define  _ArchiveComponent_ as a subtype of _CreativeWork_.
- Drop, as you have, expansion of domain for properties (_hasPart_, _isPartOf_, _location_, _itemCondition_)
- Define a new property _itemLocation_, as a sub-property of _location_, with a domain of _ArchiveComponent_ ,or possibly _CreativeWork_.
- Hold over to future discussion a replacement for the proposed expanded use of _itemCondition_
- Adjust the supplied examples to reflect these choices.

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
